src/tests/system/tests/test_ad.py

```diff
@@ -0,0 +1,54 @@
+"""
+SSSD Active Directory (AD) Test Cases
+
+:requirement: ad
+"""
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.ad import AD
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.topology(KnownTopology.AD)
+@pytest.mark.ticket(gh=7174)
+@pytest.mark.parametrize("method", ["su", "ssh"])
+@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
+@pytest.mark.importance("critical")
```

Comment:
Is this really a critical test? The different letter cases look like edge cases.

Reply:
I would argue this test case is critical, as we don't have a test using the email attribute to login.